spring-cloud-starter-stream-sink-gemfire/src/test/java/org/springframework/cloud/stream/app/gemfire/sink/normal/GemfireSinkIntegrationTests.java

```diff
@@ -56,8 +56,8 @@
 				"gemfire.pool.connectType=server",
 				"spring.cloud.stream.default.binder=test"
 		},
-        classes = {GemfireSinkConfiguration.class})
-@DirtiesContext(methodMode = DirtiesContext.MethodMode.BEFORE_METHOD)
+		classes = { GemfireSinkConfiguration.class })
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
```

Comment:
Actually it works even just with the default after class.
I mean that we don't need this extra control after each method.